saleor/graphql/menu/resolvers.py

```diff
@@ -1,10 +1,23 @@
+import graphene
+
 from ...menu import models
 from ..utils import filter_by_query_param
+from .types import Menu
 
 MENU_SEARCH_FIELDS = ('name',)
 MENU_ITEM_SEARCH_FIELDS = ('name',)
 
 
+def resolve_menu(info, id=None, name=None):
+    assert id or name, 'No ID or name provided.'
+    if name is not None:
+        try:
+            return models.Menu.objects.get(name=name)
+        except models.Menu.DoesNotExist:
+            return None
```

Comment:
Please test this. I remember I once encountered an issue, when resolver was returning `None`, counterintuitively query was returning _all_ objects of particular model.